client/src/app/core/services/pdf-document.service.ts

```diff
@@ -154,19 +139,40 @@ export class PdfDocumentService {
             },
             info: metadata,
             content: documentContent,
-            styles: style
+            styles: this.getStandardPaperStyles()
         };
+        await this.loadAllImages();
+        return result;
+    }
 
-        // add images to Vfs
-        const promises = this.images.map(image => {
-            return this.addImageToVfS(image);
-        });
-        await Promise.all(promises);
+    /**
+     * Overall document definition and styles for blank PDF documents
+     * (e.g. ballots)
+     *
+     * @param documentContent the content of the pdf as object
+     * @param imageUrl an optional image to insert into the ballot
+     * @returns the pdf document definition ready to export
+     */
+    private async getBallotPaper(documentContent: object, imageUrl?: string): Promise<object> {
+        this.initFonts();
+        this.imageUrls = imageUrl ? [imageUrl] : [];
```

Comment:
looks odd compared to `getStandardPaper`
maybe there is the error.

Reply:
nope. it was using the image url with the slash in front of the file name (which you always removed in 2.3 code, and I didn't in ballot paper cases)